### PR TITLE
[ADD] fieldservice_sale_recurring

### PR DESCRIPTION
**Pull request by max3903**
@brian10048 I need to be able to sell field services without the recurring feature.

### CODE REVIEW

File: fieldservice_sale_recurring/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
```
Comment (mgosai) [inline fieldservice_sale_recurring/models/__init__.py]: please remove 'Today'

File: fieldservice_sale_recurring/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2019 - TODAY, Brian McMaster, Open Source Integrators
```
Comment (mgosai) [inline fieldservice_sale_recurring/models/product_template.py]: same here

File: fieldservice_sale_recurring/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2019 - TODAY, Brian McMaster, Open Source Integrators
```
Comment (mgosai) [inline fieldservice_sale_recurring/models/sale_order.py]: here too

File: fieldservice_sale_recurring/models/sale_order_line.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2019 - TODAY, Brian McMaster, Open Source Integrators
```
Comment (mgosai) [inline fieldservice_sale_recurring/models/sale_order_line.py]: here also

**Review by mgosai — APPROVED** [review]

File: fieldservice_sale_recurring/models/product_template.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2019 Brian McMaster
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
```
Comment (brian10048) [inline fieldservice_sale_recurring/models/product_template.py:10]:
```suggestion

    field_service_tracking = fields.Selection(
        selection_add=[
            ('recurring', 'Create a recurring order')
        ]
    )
```

**Review by brian10048 — CHANGES_REQUESTED** [review]
@max3903 I think there is only one addition needed and this will function as it was before

**Review by brian10048 — APPROVED** [review]
Tested and working as intended

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Review by osi-scampbell — APPROVED** [review]

**Review by dreispt — COMMENTED** [review]
This is basically moving the recurring feature into it's won module, and shoudl be OK.
What about the tests - any idea why they are failing?

**Comment by max3903** [comment]
@dreispt I am working on fixing the tests here #367

**Comment by dreispt** [comment]
Recurring invoicing should be OCA/contract feature, not an OCA/field-service feature.
I think we might be mixing domains here.

**Comment by hparfr** [comment]
> Recurring invoicing should be OCA/contract feature, not an OCA/field-service feature.
> I think we might be mixing domains here.

@dreispt are you referring the module fieldservice_sale_recurring (this PR) or fieldservice_contract (linked PR) ?

**Pull request merged**